### PR TITLE
[DOCS] 8.3.1 Release notes 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/2160.

Previews:
- [Index page](https://security-docs_2163.docs-preview.app.elstc.co/guide/en/security/master/release-notes.html)
- [Release notes](https://security-docs_2163.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.3.0.html#release-notes-8.3.1)

### CODE REVIEW

File: docs/release-notes/8.3.asciidoc [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 == 8.3
 
+[discrete]
+[[release-notes-8.3.1]]
+=== 8.3.1
+
+[discrete]
+[[bug-fixes-8.3.1]]
+==== Bug fixes and enhancements
+* Fixes a bug that prevented the Cases widget in the Detection & response dashboard from updating ({pull}135128[#135128]).
```
Comment (joepeeples) [inline docs/release-notes/8.3.asciidoc:11]:
```suggestion
* Fixes a bug that prevented the Cases widget in the Detection & Response dashboard from updating ({pull}135128[#135128]).
```

**Review by joepeeples — APPROVED** [review]
Tiny edit to match UI -- LGTM!

**Review by jmikell821 — APPROVED** [review]
+1 for Joe's comment. LGTM otherwise - thanks!

**Review by benironside — APPROVED** [review]
LGTM!

**Pull request merged**